CRM/Civicase/Service/CaseCategoryFromUrl.php

```diff
@@ -142,6 +144,10 @@ private function getParamValueFromEntryUrl($param) {
   private function getCaseCategoryFromActivityIdInUrl($activityIdParamName) {
     $activityId = CRM_Utils_Request::retrieve($activityIdParamName, 'Integer');
 
+    if (!$activityId) {
```

Comment:
why is CRM_Utils_Request::retrieve is not able to get a param which you are getting through $_GET in below function

Reply:
It is because of the cast that applies the second argument. There is no 'Array' type on this method.